packages/adapters/src/adapters/Cornerstone3D/MeasurementReport.ts

```diff
@@ -371,15 +420,18 @@ export default class MeasurementReport {
             defaultState.description = defaultState.finding.CodeMeaning;
         }
 
-        defaultState.annotation.metadata.label =
+        defaultState.annotation.data.label =
             MeasurementReport.getCornerstoneLabelFromDefaultState(defaultState);
 
         return {
             defaultState,
+            state: defaultState,
```

Comment:
Why are we adding a new state which is equal to default state when a default state is already being returned?